test/Spec.hs

```diff
@@ -3,16 +3,24 @@ module Main (main) where
 import Hedgehog (Group (..), checkParallel)
 import System.IO (hSetEncoding, utf8)
 
+import CakeSlayer.Jwt (decodeIntIdPayload, decodeTextIdPayload, encodeIntIdPayload,
+                       encodeTextIdPayload)
+import Test.Gen (genInt, genText)
+import Test.Jwt (jwtRoundtrip)
 import Test.Password (pwdHashVerify)
 
 
 hedgehogTests :: Group
 hedgehogTests = Group "Roundtrip properties"
-    [ pwdHashVerify `named` "verify . hash ≡ True"
+    [ "verify  . hash        ≡ True" `named` pwdHashVerify
+    , "fromMap . toMap @Int  ≡ Just" `named`
```

Comment:
It's not clear from the description that these are JWT related tests

Reply:
Done!